src/main/java/com/maxmind/minfraud/response/Email.java

```diff
@@ -37,4 +40,14 @@ public Boolean isFree() {
     public Boolean isHighRisk() {
         return isHighRisk;
     }
+
+    /**
+     * @return A date string (e.g. 2017-04-24) to identify the date an email
+     *         address was first seen by MaxMind. This is expressed using the
+     *         ISO 8601 date format.
+     */
+    @JsonProperty("first_seen")
+    public String firstSeen() {
```

Comment:
This should be `getFirstSeen()` to follow the JavaBeans naming conventions used throughout. A `Date` might be a bit nicer, but it looks like we don't do that for `/device/last_seen` (probably for time zone reasons there) so maybe a string is best.